code/modules/research/message_server.dm

```diff
@@ -102,7 +103,10 @@ var/global/list/obj/machinery/message_server/message_servers = list()
 	//	decryptkey = generateKey()
 	if(active && (stat & (BROKEN|NOPOWER)))
 		active = 0
-		return
+	else if(toggled)
+		active = 1
+	else

```

Comment:
No. I've tested it and it works as intended. The PDA server is initially on, it can be turned off, and it can be turned back on. It works perfectly.